src/io/flutter/logging/FlutterLogFilterPanel.form

```diff
@@ -1,25 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="io.flutter.logging.FlutterLogFilterPanel">
-  <grid id="27dc6" binding="root" layout-manager="GridLayoutManager" row-count="1" column-count="4" same-size-horizontally="false" same-size-vertically="false" hgap="3" vgap="-1">
+  <grid id="27dc6" binding="root" layout-manager="GridLayoutManager" row-count="1" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
     <margin top="0" left="0" bottom="0" right="0"/>
     <constraints>
-      <xy x="20" y="20" width="755" height="44"/>
+      <xy x="20" y="34" width="450" height="35"/>
     </constraints>
     <properties>
+      <background color="-1250068"/>
       <enabled value="true"/>
-      <preferredSize width="986" height="35"/>
+      <minimumSize width="450" height="35"/>
+      <preferredSize width="450" height="35"/>
     </properties>
     <border type="none"/>
     <children>
-      <grid id="9d27c" binding="panelExpression" layout-manager="GridLayoutManager" row-count="1" column-count="1" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```

Comment:
I remove `panelExpression` which is the container of `expressionTextField`.
Instead, I put `expressionTextField` directly in the layout.